pkgs/development/ruby-modules/bundler-env/basic.nix

```diff
@@ -0,0 +1,140 @@
+{ stdenv, runCommand, ruby, lib
+, defaultGemConfig, buildRubyGem, buildEnv
+, makeWrapper
+, bundler
+}@defs:
+
+{
```

Comment:
it would be helpful to describe the responsibilities of the function since it's so complicated